code/src/pages/MoviesHome.js

```diff
@@ -0,0 +1,35 @@
+import React, { useState, useEffect } from 'react'
+import { MovieInfo } from 'components/MovieInfo'
+
+
+import { apiKey } from "key"
+import "./moviesHome.css"
+import "index.css"
+
+export const MoviesHome = () => {
+    const [movies, setMovies] = useState([])
+    const [loading, setLoading] = useState(true)
+
+    useEffect(() => {
+        const URL = `https://api.themoviedb.org/3/movie/popular?api_key=${apiKey}&language=en-US&page=1`
+        fetch(URL)
+            .then((res) => res.json())
+            .then((json) => {
+                setLoading(false)
+                setMovies(json.results)
```

Comment:
By mistake you have created infinite loop of requests, check out your network tab :)
It's because of `loading` state property being present in dependency array in line 24. Consider deleting it from there